cms/templates/js/highlights-editor.underscore

```diff
@@ -1,5 +1,33 @@
 <form>
 <h3 class="modal-section-title" id="highlights_label"><%- gettext('Section Highlights') %></h3>
+    <div class="highlights-info">
+
+<% if (highlights_preview_only) { %>
+    <p><b>
+    <%- gettext('This feature is currently in testing. Course teams can enter highlights, but learners will not receive email messages.') %>
```

Comment:
@mmacfarlane - we had to rush this into the release. Here is the text that will go out Monday morning to course teams that don't have the emails enabled in their course.

Does this sound OK?

Reply:
@mulby Looks good. Thanks for getting that in.